src/Makefile.qt.include

```diff
@@ -445,9 +445,9 @@ $(srcdir)/qt/bitcoinstrings.cpp: $(libdogecoin_server_a_SOURCES) $(libdogecoin_w
 	@test -n $(XGETTEXT) || echo "xgettext is required for updating translations"
 	$(AM_V_GEN) cd $(srcdir); XGETTEXT=$(XGETTEXT) PACKAGE_NAME="$(PACKAGE_NAME)" COPYRIGHT_HOLDERS="$(COPYRIGHT_HOLDERS)" COPYRIGHT_HOLDERS_SUBSTITUTION="$(COPYRIGHT_HOLDERS_SUBSTITUTION)" $(PYTHON) ../share/qt/extract_strings_qt.py $^
 
-translate: $(srcdir)/qt/bitcoinstrings.cpp $(QT_FORMS_UI) $(QT_FORMS_UI) $(BITCOIN_QT_BASE_CPP) qt/bitcoin.cpp $(BITCOIN_QT_WINDOWS_CPP) $(BITCOIN_QT_WALLET_CPP) $(BITCOIN_QT_H) $(BITCOIN_MM)
+translate: $(srcdir)/qt/bitcoinstrings.cpp $(QT_FORMS_UI) $(BITCOIN_QT_BASE_CPP) qt/bitcoin.cpp $(BITCOIN_QT_WINDOWS_CPP) $(BITCOIN_QT_WALLET_CPP) $(BITCOIN_QT_H) $(BITCOIN_MM)
 	@test -n $(LUPDATE) || echo "lupdate is required for updating translations"
-	$(AM_V_GEN) QT_SELECT=$(QT_SELECT) $(LUPDATE) $^ -locations relative -no-obsolete -ts $(srcdir)/qt/locale/bitcoin_en.ts
+	$(AM_V_GEN) QT_SELECT=$(QT_SELECT) $(LUPDATE) $^ -locations relative -ts $(srcdir)/qt/locale/bitcoin_en.ts
```

Comment:
running `make translate` only updates `bitcoin_en.ts`:

```console
$ make -C src translate 
  GEN      qt/bitcoinstrings.cpp
  GEN      translate
Updating 'qt/locale/bitcoin_en.ts'...
    Found 1073 source text(s) (2 new and 1071 already existing)
    Kept 4 obsolete entries
```

We basically need 2 targets (unlike Bitcoin Core as they use Transifex):

1. Update bitcoin_en.ts (maintainer, standard operation when strings change, simply so that we have a translation file for comparison)
2. Update a particular translation's base strings (contributor, before updating a translation file)

With the below suggestion, this would mean something like this:

```console
$ make -C src translate_base
make: Entering directory '/home/patrick/dogecoin-localbuild/src'
  GEN      translate_base
Updating 'qt/locale/bitcoin_en.ts'...
    Found 1073 source text(s) (2 new and 1071 already existing)
    Removed 4 obsolete entries
make: Leaving directory '/home/patrick/dogecoin-localbuild/src'
$ make -C src translate language=de
make: Entering directory '/home/patrick/dogecoin-localbuild/src'
  GEN      translate
Updating 'qt/locale/bitcoin_de.ts'...
    Found 1073 source text(s) (2 new and 1071 already existing)
    Kept 4 obsolete entries
make: Leaving directory '/home/patrick/dogecoin-localbuild/src'
$ make -C src translate language=en
make: Entering directory '/home/patrick/dogecoin-localbuild/src'
cannot translate English, use make translate_base
make: *** [Makefile:11217: translate] Error 1
make: Leaving directory '/home/patrick/dogecoin-localbuild/src'
$ make -C src translate
make: Entering directory '/home/patrick/dogecoin-localbuild/src'
you need to specify a language
make: *** [Makefile:11216: translate] Error 1
make: Leaving directory '/home/patrick/dogecoin-localbuild/src'
```

```suggestion
translate_base: $(srcdir)/qt/bitcoinstrings.cpp $(QT_FORMS_UI) $(BITCOIN_QT_BASE_CPP) qt/bitcoin.cpp $(BITCOIN_QT_WINDOWS_CPP) $(BITCOIN_QT_WALLET_CPP) $(BITCOIN_QT_H) $(BITCOIN_MM)
  @test -n $(LUPDATE) || echo "lupdate is required for updating translations"
  $(AM_V_GEN) QT_SELECT=$(QT_SELECT) $(LUPDATE) $^ -locations relative -ts $(srcdir)/qt/locale/bitcoin_en.ts
  $(AM_V_GEN) QT_SELECT=$(QT_SELECT) $(LUPDATE) $^ -locations relative -no-obsolete -ts $(srcdir)/qt/locale/bitcoin_en.ts

translate:
  @test -n $(LUPDATE) || echo "lupdate is required for updating translations"
  @if [ "$(language)" = "" ]; then echo "you need to specify a language" && exit 1; fi
  @if [ "$(language)" = "en" ]; then echo "cannot translate English, use make translate_base" && exit 1; fi
  $(AM_V_GEN) QT_SELECT=$(QT_SELECT) $(LUPDATE) $^ -ts $(srcdir)/qt/locale/bitcoin_$(language).ts
```

Reply:
Ah, I see what you meant. I've added a change like this and updated the doc to reflect it.

Reply:
Cool, thanks! 

I think that for consistency with how the current translation files work, we should omit `-locations relative` from the actual `translate` target (but keep it for `translate_base`) 

I'd only remove `-no-obsolete` for `translate` as well: It makes no sense to keep it for `translate_base` because there is no manual translation happening there.

Reply:
I'm not sure I'm interpreting your `-no-obsolete` suggestion the way you intended, so I've pushed one more commit so that we can discuss this with code.

I'll probably end up squashing that commit into another before it's time to merge.

Reply:
👍 This is exactly what I meant.